namespace_registry/registry_db.go

```diff
@@ -78,7 +80,7 @@ func createNamespaceTable() {
         prefix TEXT NOT NULL UNIQUE,
         pubkey TEXT NOT NULL,
         identity TEXT,
-        admin_metadata TEXT
+        admin_metadata TEXT CHECK (length("admin_metadata") <= 4000)
```

Comment:
Is there some kind of logic behind restricting the length of what admins can set in the `admin_metadata` column to 4000? If so, could you add a comment?

Reply:
Will do, but basically admin_metadata might not necessarily just mean data set by the admins. We might eventually add things such as "creator" or something else for example, which could be determined by the person who creates the namespace. This is to put a hard limit on what is written to disc to prevent someone maliciously potentially inserting something here of a huge length.